syntax/snakemake.vim

```diff
@@ -30,16 +30,16 @@ source $VIMRUNTIME/syntax/python.vim
 syn keyword pythonStatement	include workdir onsuccess onerror
 syn keyword pythonStatement	ruleorder localrules configfile
 syn keyword pythonStatement	touch protected temp wrapper
-syn keyword pythonStatement	input output params message threads resources
-syn keyword pythonStatement	version run shell benchmark snakefile log script
-syn keyword pythonStatement	rule subworkflow nextgroup=pythonFunction skipwhite
+syn keyword pythonStatement	input output params message threads priority resources wildcard_constraints
+syn keyword pythonStatement	version run shell benchmark snakefile log script shadow group cwl
+syn keyword pythonStatement	rule subworkflow checkpoint nextgroup=pythonFunction skipwhite
```

Comment:
I tried to group the new keywords together with similar keywords, I hope that I got it somewhat right